src/Microsoft.Health.Fhir.CosmosDb/Features/Search/Queries/QueryBuilder.cs

```diff
@@ -105,42 +108,22 @@ public SqlQuerySpec GenerateHistorySql(SearchOptions searchOptions)
 
             private void AppendSelectFromRoot(string selectList = SearchValueConstants.RootAliasName)
             {
-                _queryBuilder
-                    .Append("SELECT ")
-                    .Append(selectList)
-                    .Append(" FROM root ")
-                    .AppendLine(SearchValueConstants.RootAliasName);
+                _queryHelper.AppendSelectFromRoot(selectList);
```

Comment:
Does it make sense to still have these private methods since they simply call the query helper? Maybe we should just call the query helper directly.

Reply:
The private methods have defaults. We will have to add values everywhere since we will different values for controlplane vs fhir.